playbook-website/app/javascript/site_styles/_landing-page.scss

```diff
@@ -79,4 +83,17 @@
       }
     }
   }
+
+  // Social proof scss
+
+  .banner-background-width {
+    width: 100%;
+  }
```

Comment:
We have a story for this that is separate for adding a global width prop. 
The plan is to ship this, and then we will come back and this will be the first place we test our new global width prop. Then we will promptly remove this custom css.